assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -83,7 +83,7 @@ const DateHelperText = styled(Text).attrs(() => ({
 `;
 // TODO: Fix dirty workaround.
 const ListIcon = styled(LockSVG)`
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray900};
+  color: ${({ theme }) => theme.colors.gray[90]};
```

Comment:
after 
![after](https://user-images.githubusercontent.com/41136059/113447719-44eee980-93af-11eb-90c3-89808d51d898.png)

Reply:
Nice! I think we may be able to make this `fg.primary` if we want